crates/primitives/src/forks.rs

```diff
@@ -4,8 +4,12 @@ use sov_rollup_interface::fork::{fork_pos_from_block_number, verify_forks, Fork}
 use sov_rollup_interface::spec::SpecId;
 use sov_rollup_interface::Network;
 
+#[cfg(not(feature = "testing"))]
 static FORKS: OnceLock<&'static [Fork]> = OnceLock::new();
 
+#[cfg(feature = "testing")]
+pub static FORKS: OnceLock<&'static [Fork]> = OnceLock::new();
+
```

Comment:
if this is made public for tests, then we can remove the not(testing) (testing) separation in the use_network_forks as it will no longer be used for testing